ee/server/service/teams.go

```diff
@@ -975,6 +978,7 @@ func (svc *Service) createTeamFromSpec(
 				WindowsUpdates:       spec.MDM.WindowsUpdates,
 				MacOSSettings:        macOSSettings,
 				MacOSSetup:           macOSSetup,
+				WindowsSettings:      spec.MDM.WindowsSettings,
```

Comment:
Looks like this was never set on `Apply teams` when the team was getting created. It probably did not matter much because on the next run of the same `fleetctl apply` (or `gitops`), it would've been an update and would've been set anyway.